torch/_inductor/codegen/wrapper.py

```diff
@@ -2086,6 +2086,14 @@ def codegen_subgraph(self, subgraph, outer_inputs, outer_outputs):
 
         self.codegen_subgraph_suffix(subgraph, outer_inputs, outer_outputs)
 
+    def codegen_invoke_subgraph(self, invoke_subgraph):
```

Comment:
Should we add this to the `cpp_wrapper_cpu` with `NotImplementedException`?

Reply:
Good catch.